src/utils/errors/constants.ts

```diff
@@ -0,0 +1,92 @@
+import { CustomErrorMapping } from "./types";
+
+export const STRING_PREFIX_AFTER_UNICODE_REPLACEMENT = 'y %';
```

Comment:
Do we know what this means? Looking at the usage, I don't understand how the name of the constant here makes sense (I know you just migrated it from what it historically was)

Reply:
given the case that we keep it, a different name would be better IMHO,  `SANITIZED_MESSAGE_SEPARATOR` or similar

but now that I've mentioned, I think we could remove this and be shortened to something that looks like:  

```
const errorMessage = sanitizeAscii(cause.message);
```

since we're already properly sanitizing ASCII, and we don't need *that much* string manipulation besides this right??

Reply:
Yeah reading again I think either this string used to be placed somehow in the string or it's an artefact from something that emitted errors with that string, and someone replaced it. Anyhow, the purpose is unclear to me. Maybe @zy0n or @mattgle remember seeing this string in errors at some point?

Reply:
I have no idea tbh, but @mesquka might know